Can't use absolute module names in Jest

## Prerequisites

- [x] Using npm
- [x] Using an up-to-date [`main` branch](https://github.com/electron-react-boilerplate/electron-react-boilerplate/tree/main)
- [x] Using latest version of devtools. [Check the docs for how to update](https://electron-react-boilerplate.js.org/docs/dev-tools/)
- [x] Tried solutions mentioned in [#400](https://github.com/electron-react-boilerplate/electron-react-boilerplate/issues/400)
- [x] For issue in production release, add devtools output of `DEBUG_PROD=true npm run build && npm start`

## Expected Behavior

When importing `renderer` modules using TypeScript absolute imports (eg: `import hello from 'renderer/hello'`), Jest should resolve these modules during test execution.

## Current Behavior

Jest can't find any `renderer/*` modules.

## Steps to Reproduce

Given these files:

```ts
// src/renderer/hello.ts
export default function hello(name: string) {
  return `Hello ${name}!`;
}


// src/__tests__/hello.test.ts
import hello from 'renderer/hello';

it('says hello', () => {
  expect(hello('World')).toEqual('Hello World!');
});
```

Running Jest throws an error:

```
$ npm test
> test
> jest

 FAIL  src/__tests__/hello.test.ts
  ● Test suite failed to run

    Cannot find module 'renderer/hello' from 'src/__tests__/hello.test.ts'

    > 1 | import hello from 'renderer/hello';
        | ^
      2 |
      3 | it('says hello', () => {
      4 |   expect(hello('World')).toEqual('Hello World!');

      at Resolver._throwModNotFoundError (node_modules/jest-resolve/build/resolver.js:491:11)
      at Object.<anonymous> (src/__tests__/hello.test.ts:1:1)
 ```

## Possible Solution (Not obligatory)

Add `src` to `jest.moduleDirectories` in the root `package.json`.

## Context

It feels more natural to use absolute imports as we already can in any renderer modules.

## Your Environment

- Node version : v16.11.1
- electron-react-boilerplate version or branch : main (commit b4cc4ecc784bd3467b5462a33d2f854d53d8cb0b)
- Operating System and version : Ubuntu 22.04.1 LTS

--- a/package.json
+++ b/package.json
@@ -80,7 +80,8 @@
   "jest": {
     "moduleDirectories": [
       "node_modules",
-      "release/app/node_modules"
+      "release/app/node_modules",
+      "src"
     ],
     "moduleFileExtensions": [
       "js",

